content/projects/tdd/string-calculator/_index.md

```diff
@@ -6,10 +6,107 @@ ready: true
 
 Before you commence, first read through {{% contentlink "/topics/data_validation_and_verification/" %}}. Upon completion, read through {{% contentlink "/topics/regular-expressions/" %}}.
 
-#### Project
-Do this: https://osherove.com/tdd-kata-1/
+Please note that this project should be done in a TDD manner. 
 
-Do all the steps (1 - 9).
+#### 1. Create an add function that can handle up to two integers passed in as a string.
 
-### why is this important?
-- If you're wondering to yourself, "Why is this sooo important!?" take a look at {{% contentlink "/topics/data_validation/" %}}
+Create a function called `add()` that takes in a string as a parameter and behaves in the following way: 
+
+
+```
+add("")
+//should return 0
+
+add("1")
+//should return 1
+
+add("1,1")
+//should return 2
+
+```
+*Note: The output that the function returns should be an integer and not a string.*
+
+#### 2. Modify the add function to handle multiple integers.
+
+```
+add("1,2,3,4")
+//should return 10
+
+add("")
+//should still return 0
+
+add("1")
+//should still return 1
+
+add("1,1")
+//should still return 2
+```
+As you keep adding more functionality to your code always make sure that the previous functionality you implemented still works flawlessly. Keep this in mind as you continue to modify your code. 
+
+#### 3. Modify the add function so that it can handle new lines between integers.
+
+```
+add(“1\n2,3” )
```

Comment:
The whole string should be green in colour. Use straight double quotes instead of opening and closing curly double quotes.